src/components/ManaCalculator/hooks/useResultsPerEpoch.ts

```diff
@@ -40,11 +44,16 @@ export function useResultsPerEpoch(state: ManaCalculatorProps): EpochReward[] {
       i,
     );
 
-    const mana = generatedMana + passiveRewards;
+    const mana = manaGenerated + passiveRewards;
+
+    const additionalTPS = calculateTPS(passiveRewards, state.congestion);
+    const grantedTPS = calculateTPS(manaGenerated, state.congestion);
```

Comment:
I would probably rename these variables to avoid confusion, both TPS are "granted", they just come from different values of Mana, your generated mana or your pasive mana